src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -49,8 +49,8 @@
     <PackageReference Include="Microsoft.Azure.AppService.Proxy.Client" Version="2.0.11020001-fabe022e" />
     <PackageReference Include="Microsoft.Azure.Functions.JavaWorker" Version="1.8.2-SNAPSHOT" />
     <PackageReference Include="Microsoft.Azure.Functions.NodeJsWorker" Version="2.1.2" />
-    <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker.PS6" Version="3.0.630" />
     <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker.PS7" Version="3.0.912" />
```

Comment:
I think we need to replace the PS7 version with the one built from the v3.x/ps7.0 branch (which should not deploy worker.config.json to the parent folder anymore). Otherwise, we'll run into the conflict of two worker.config.json files - this is what we discussed earlier today.

Reply:
Good point. I will fix this.

Reply:
Fixed.